src/components/UI/Tables/ProductsTable/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Avatar, IconButton, TableCell } from "@mui/material";
+import { IProduct } from "shared/models";
```

Comment:
No need in shared folder

Reply:
can i remove it later it might create some big conflicts?

Reply:
Yes, So add TODO